zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/monitoring/BackupMXBean.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zookeeper.server.backup.monitoring;
+
+/**
+ * ZK backup MBean
+ */
+public interface BackupMXBean {
+  // Snapshot backup metrics
+
+  /**
+   * Counter
+   * @return Number of snapshot backup errors since last successful snapshot backup iteration
+   */
+  int getSuccessiveSnapshotIterationErrorCount();
```

Comment:
Synced with @mgao0 offline. What this metric represents is the number of consecutive failed iterations. This metric could be used for alerts - e.g.) if the count is greater than 3, alert the administrator because the error is no longer transient.

I'd like to suggest renaming this metric for both snapshot and txlogs - could we do something like `numConsecutiveFailed(Snapshot/Log)Iterations` and update the JavaDoc accordingly with a clear use case example?